unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/Tests/ProfanityWordProviderInMemory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+
+public class ProfanityWordProviderInMemory : IProfanityWordProvider
```

Comment:
I see this provider is only used in tests. We should just mock it there and avoid the implementation until we really need it.

Just from the top of my head, something like this:
```
List<string> words;
var mock = Substitute.For<IProfanityWordProvider>();
mock.Configure().GetAll().Returns(x => words);
```

Reply:
Yeah that was my first approach but i was having compilation issues after importing the dll. I will take another look